.github/workflows/test.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       - run: |
           npm run build && npm test
         env:
-          INPUT_BASEURL: https://api.getport.io
-          INPUT_CLIENTID: ${{ secrets.PORT_INTEG_CLIENT_ID }}
-          INPUT_CLIENTSECRET: ${{ secrets.PORT_INTEG_CLIENT_SECRET }}
+          PORT_BASE_URL: ${{ secrets.PORT_BASE_URL }}
+          INPUT_BASEURL: ${{ secrets.PORT_BASE_URL }}
```

Comment:
what's the difference between those 2?

Reply:
The input one gets overriden in the tests, the other one is more persistent
The tests override the actualy value of the environment variable, and I need the value of this after the tests which overrides the environment variable